src/pages/console/uibuilder/responsive.mdx

```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
 }
 ```
 
-Configure your Figma components with variants with names that match the breakpoint values. In the example below, our `TopBar` component has the following variants:
-- `variation: default`
-- `variation: small`
+Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file.  You can customize these breakpoints using the Amplify Theme Editor plugin:
+1. Right click in your Figma project and select "Plugins" > "AWS Amplify Theme Editor"
+![Context menu showing how to view plugins and select the AWS Amplify Theme Editor](/images/studio/responsive/theme-editor.png)
+
+2. Select the "Breakpoints" Tab and click on any breakpoint to customize the pixel value
+![AWS Amplify Theme Editor with the Breakpoints tab selected](/images/studio/responsive/breakpoints.png)
+
+The grey bar within the outlined box will indicate your pixel value compared to a standard 1920px FHD resolution.
+
+## 2. Configure your Figma variants
 
-The small variation should be shown when the screen size falls below the "small" breakpoint.
+To make a component responsive, create different variants for the component based on the breakpoints.  [Learn more about creating variants in Figma's documentation](https://help.figma.com/hc/en-us/articles/360061175334-Create-interactive-components-with-variants).
 
 <Callout warning>
 
-**IMPORTANT:** The variants are required to have the same component structure. If the variants don't have the same child elements, then Amplify Studio will not be able to import the component.
+**IMPORTANT:** The variants are required to have the same <strong>component structure</strong>. Elements within the component structure should use the same <strong>layout method</strong> (i.e. auto or manual). 
 
-![Image showing that the component structure needs to be the same](/images/studio/responsive/component-structure.png)
+If the variants don't have the same structure, then Amplify Studio will not be able to import the component.
+
+To customize child elements in variants, you can use the eye icon to hide and show elements in different variants. 
+
+![Image showing that the component structure needs to be the same](/images/studio/responsive/component-structure2.png)
```

Comment:
If you're changing the image, you should maybe just remove the older image. Is the other image used somewhere else?

Reply:
Older file has been removed!